specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/transfers.json

```diff
@@ -0,0 +1,1182 @@
+      "type": "object",
+      "description": "Request parameters to initiate transfer.",
+      "properties": {
+        "properties": {
+          "description": "Request parameters to initiate transfer.",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/InitiateTransferProperties"
+        }
+      }
+    },
+    "InitiateTransferProperties": {
+      "type": "object",
+      "description": "Request parameters to initiate transfer.",
+      "properties": {
+        "recipientEmailId": {
+          "type": "string",
+          "description": "The email ID of the recipient to whom the transfer request is sent."
+        }
+      }
+    },
+    "PartnerInitiateTransferRequest": {
+      "type": "object",
+      "description": "Request parameters to initiate partner transfer.",
+      "properties": {
+        "properties": {
+          "description": "Request parameters to initiate partner transfer.",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/PartnerInitiateTransferProperties"
+        }
+      }
+    },
+    "PartnerInitiateTransferProperties": {
+      "type": "object",
+      "description": "Request parameters to initiate transfer.",
+      "properties": {
+        "recipientEmailId": {
+          "type": "string",
+          "description": "The email ID of the recipient to whom the transfer request is sent."
+        },
+        "resellerId": {
+          "type": "string",
+          "description": "Optional MPN ID of the reseller for transfer requests that are sent from a Microsoft Partner Agreement billing account."
+        }
+      }
+    },
+    "AcceptTransferRequest": {
+      "type": "object",
+      "description": "Request parameters to accept transfer.",
+      "properties": {
+        "properties": {
+          "description": "Request parameters to accept transfer.",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/AcceptTransferProperties"
+        }
+      }
+    },
+    "AcceptTransferProperties": {
+      "type": "object",
+      "description": "Request parameters to accept transfer.",
+      "properties": {
+        "productDetails": {
+          "description": "Request parameters to accept transfer.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ProductDetails"
+          }
+        }
+      }
+    },
+    "ProductDetails": {
+      "type": "object",
+      "description": "Details of the product that is transferred.",
+      "properties": {
+        "productType": {
+          "description": "Type of the product that is transferred.",
+          "$ref": "#/definitions/ProductType"
+        },
+        "productId": {
+          "type": "string",
+          "description": "The ID of the product that is transferred."
+        }
+      }
+    },
+    "ProductType": {
+      "type": "string",
+      "description": "The type of product that is transferred.",
+      "enum": [
+        "AzureSubscription",
```

Comment:
Can we add `Unknown` as a fallback in case additional enum values get added in future versions?

Same goes for other enums in this file.